scripts/west_commands/runners/jlink.py

```diff
@@ -69,6 +76,12 @@ def do_add_parser(cls, parser):
                             help='J-Link Commander, default is JLinkExe')
         parser.add_argument('--erase', default=False, action='store_true',
                             help='if given, mass erase flash before loading')
+        parser.add_argument('--reset-after-load', '--no-reset-after-load',
+                            dest='gen_reset_after_load', nargs=0,
+                            action=ToggleAction,
+                            help='''reset after loading? (default: no)''')
```

Comment:
No need for triple quoting of a single line string; `'reset after loading? (default: no)'` will do.

Reply:
Ok